flow/project.py

```diff
@@ -2510,14 +2548,16 @@ def _get_submission_operations(self, jobs, default_directives, names=None,
                                    ignore_conditions=IgnoreConditions.NONE,
                                    ignore_conditions_on_execution=IgnoreConditions.NONE):
         """Grabs JobOperations that are eligible to run from FlowGroups."""
-        for job in jobs:
-            for group in self._gather_flow_groups(names):
-                if group.eligible(job, ignore_conditions) and self._eligible_for_submission(group,
-                                                                                            job):
+        for group in self._gather_flow_groups(names):
+            for job in jobs:
```

Comment:
`job` should be a list of jobs here.

Reply:
The `aggregate` parameter, when associated with a group, will have that behaviour. 
But for now, I think passing [job] in should be enough.